decidim-debates/app/helpers/decidim/debates/application_helper.rb

```diff
@@ -35,7 +35,7 @@ def text_editor_for_debate_description(form)
       def filter_origin_values
         origin_values = []
         origin_values << TreePoint.new("official", t("decidim.debates.debates.filters.official"))
-        origin_values << TreePoint.new("citizens", t("decidim.debates.debates.filters.citizens"))
+        origin_values << TreePoint.new("participants", t("decidim.debates.debates.filters.participants"))
```

Comment:
The I18n keys need to be changed. 
from `decidim.debates.debates.filters.citizens` to `decidim.debates.debates.filters.participants` in the decidim-debates/config/locales
Or you may remove them

Reply:
Already answered here: https://github.com/decidim/decidim/pull/8697#discussion_r783005552